mdflynn/ideabox-boilerplate
# Aligned header menu and title text

## What is the change?

This adds styling to the header menu and title text in mobile view.

## What does it fix?

This fixes the issue of our sections not stacking on each other and formatting. 

## Is this a fix or a feature? 

This is both a fix and a feature. It fixes our meta tag issue that was not allowing us to design our CSS. And this is a feature because it correctly displays the header and title. 

## Where should the reviewer start?

The reviewer should start at line 80 in the CSS underneath the media query.

## How should this be tested?

Open the Index.html file. Then open the Chrome dev tools and change to a mobile view to see our new changes.
